flow/activities/flowable.go

```diff
@@ -816,6 +815,12 @@ func (a *FlowableActivity) RecordSlotSizes(ctx context.Context) error {
 				attribute.String(otel_metrics.FlowStatusKey, status.String()),
 				attribute.Bool(otel_metrics.IsFlowActiveKey, info.isActive),
 			)))
+
+			if flowMetadata.Status == protos.FlowStatus_STATUS_COMPLETED ||
+				flowMetadata.Status == protos.FlowStatus_STATUS_TERMINATED {
```

Comment:
Not sure if this will be populated, on phone rn. There's also status variable we are using above